repos/system_upgrade/common/models/hybridimage.py

```diff
@@ -0,0 +1,11 @@
+from leapp.models import Model
+from leapp.topics import SystemFactsTopic
+
+
+class HybridImage(Model):
```

Comment:
Is there any chance this can be used for non-azure images? If not - maybe it's worth renaming it to `HybridImageAzure` to make it clear that is for azure only?

Reply:
good idea! changed!